cfpb/cfgov-refresh
# Redirects for /educational-resources

We are planning to change the  `/educational-resources` section in Wagtail to `/practitioner-resources` and move two of its subpages under `/consumer-tools`. This adds redirects to the new page locations.

## Additions

- Educational resource redirects


## Testing

To test redirects locally,
1. Change slug of [educational-resources](http://localhost:8000/admin/pages/1180/edit/) section to `practitioner-resources` in admin.
2. Move `practitioner-resources/money-as-you-grow/` under `consumer-tools`.
3. Unpublish   `practitioner-resources/resources-for-older-adults/managing-someone-elses-money/ `and publish the new `/consumer-tools/managing-someone-elses-money/` draft page.
4. Verify educational resources page redirects: [youth-financial-education](http://localhost:8000/educational-resources/youth-financial-education/), [money as you grow](http://localhost:8000/educational-resources/money-as-you-grow/)


## Checklist
